src/bundle.c

```diff
@@ -682,7 +682,9 @@ static int install_bundles(struct list *bundles, struct list **subs, int current
 	grabtime_start(&times, "Add bundles and recurse");
 	ret = add_subscriptions(bundles, subs, current_version, mom, false, 0);
 
-	if (ret != add_sub_NEW) {
+	/* Use a bitwise AND with the add_sub_NEW mask to determine if at least
+	 * one new bundle was subscribed */
+	if (!(ret & add_sub_NEW)) {
```

Comment:
lgtm, but this is going to break a test. I've seen that you fixed that in another commit, but It's nice to have the broken test fixed in the same commit that breaks it.